gamemodes/terrortown/gamemode/shared/lang/en.lua

```diff
@@ -1087,17 +1087,17 @@ L.menu_administration_title = "Administration"
 L.menu_equipment_title = "Edit Equipment"
 L.menu_shops_title = "Edit Shops"
 
-L.menu_changelog_description = "A list of changes and fixes in recent versions"
-L.menu_guide_description = "Helps you to get started with TTT2 and explains some things about gameplay, roles and other stuff"
-L.menu_bindings_description = "Bind specific features of TTT2 and its addons to your own liking"
-L.menu_language_description = "Select the language of the game"
-L.menu_appearance_description = "Tweak the appearance and performance of the UI"
-L.menu_gameplay_description = "Avoid roles and tweak some features"
-L.menu_addons_description = "Configure local addons to your liking"
-L.menu_legacy_description = "A panel with converted tabs from original TTT, they should be ported over to the new system"
-L.menu_administration_description = "General settings for HUDs, shops etc"
-L.menu_equipment_description = "Set credits, limitations, availability and other stuff"
-L.menu_shops_description = "Add/Remove shops to roles and define the equipment in it"
+L.menu_changelog_description = "A list of changes and fixes in recent versions."
+L.menu_guide_description = "Helps you to get started with TTT2 and explains some things about gameplay, roles and other stuff."
+L.menu_bindings_description = "Bind specific features of TTT2 and its addons to your own liking."
+L.menu_language_description = "Select the language of the gamemode."
+L.menu_appearance_description = "Tweak the appearance and performance of the UI."
+L.menu_gameplay_description = "Avoid roles and tweak some features."
+L.menu_addons_description = "Configure local addons to your liking."
+L.menu_legacy_description = "A panel with converted tabs from the original TTT that should be ported over to the new system."
+L.menu_administration_description = "General settings for HUDs, shops, etc."
+L.menu_equipment_description = "Set credits, limitations, availability, and other stuff."
```

Comment:
same as the other comment regarding the oxford comma

Reply:
I think it should be used since it was used almost in every project where I was translating something. I have also seen that many study books and mass media use it, but not every, of course.

Reply:
The comma is now removed.